src/lib.rs

```diff
@@ -97,8 +97,11 @@
 //! Lines not mentioned are not tested: for example, the above file does not state whether the
 //! `Compiler`s `stdout` should have content or not (but note that the line `stdout:` on its own
 //! would state that the `Compiler` should have no content at all). `stderr`/`stdout` tests can use
-//! `...` as a simple wildcard: if a line consists solely of `...`, it means either "match zero or
-//! more lines"; if a line begins with `...`, it means "match the remainder of the line only".
+//! `...` as a simple wildcard: if a line consists solely of `...`, it means "match zero or
+//! more lines"; if a line begins with `...`, it means "match the remainder of the line only";
+//! if a line ends with `...`, it means "match the start of the line only". A line may start and
+//! end with `...`. `stderr`/`stdout` matches ignore leading/trailing whitespace and newlines, but
+//! are case sensitive.
```

Comment:
Lets add an example. It's not too hard to understand but it's the first thing people jump to when they see an intimidating block of documentation :P